server/config/config.go

```diff
@@ -862,6 +865,8 @@ func (c *ScheduleConfig) adjust(meta *configMetaData, reloading bool) error {
 	if !meta.IsDefined("enable-joint-consensus") {
 		c.EnableJointConsensus = defaultEnableJointConsensus
 	}
+	// we don't want user to change it at any time, it's used by tools like BR
+	c.EnableTiKVSplitRegion = true
```

Comment:
IMO, it's better to keep consistent with other configs. It can be modified through the config file at the beginning.

Reply:
https://github.com/tikv/pd/pull/5485/commits/714095cd5bb2f0361246dc07fd371283785db98b